typescript/infra/scripts/module-can-verify.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  ChainMap,
+  HyperlaneCore,
+  moduleCanCertainlyVerify,
+} from '@hyperlane-xyz/sdk';
+import { Address, ProtocolType } from '@hyperlane-xyz/utils';
+
+import { deployEnvToSdkEnv } from '../src/config/environment';
+
+import { getArgs, getEnvironmentConfig } from './utils';
+
+const mainnetHyperlaneDefaultIsmCache: ChainMap<Address> = {
+  ethereum: '0x3Ef03aEf1392E5e0C16fd4D22C3c3b4f81C8AF0C',
```

Comment:
Would prefer we not keep consts like this in a script file. Can we find it a home in `/config/` somewhere?